src/pages/Popup/Popup.jsx

```diff
@@ -1,25 +1,54 @@
-import React from 'react';
-import logo from '../../assets/img/logo.svg';
-import Greetings from '../../containers/Greetings/Greetings';
+import React, {useEffect, useState} from 'react';
+import { createQAUrl } from '../Content/utiles/utiles.js';
 import './Popup.css';
 
+//Inject the content-script to tab
+function injectContentScript(tabId){
+    chrome.scripting.executeScript({
+      files: ['contentScript.bundle.js'],
+      target: { tabId },
+    });
+}
+
 const Popup = () => {
+
+  const [status, setStatus] = useState("initializing");
+  const [url, setUrl] = useState();
+
+  //Set statue to "redirect" if the current tab url doesn't contain "isqa=true", otherwise inject the contentScript to the current tab.
```

Comment:
```suggestion
  // Set status to "redirect" if the current tab url doesn't contain "isqa=true", otherwise inject the contentScript to the current tab.
```